packages/x-charts-pro/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-alpha.0",
+  "version": "7.0.0",
```

The changelog says that it is not and we also agreed during the meeting that the pre-release phase is expected and decided upon. 👌
```suggestion
  "version": "7.0.0-alpha.0",
```